### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by distantnative**
### Enhancements
- Upgraded dependencies

## Breaking changes

None

## Ready?

- [ ] Unit tests for fixed bug/feature
- [ ] In-code documentation (wherever needed)
- [ ] CI checks pass

## When merging

- [ ] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)
- [x] Add changes to release notes draft in Notion

### CODE REVIEW

File: panel/package.json [hunk 1]
```diff
@@ -20,13 +20,13 @@
...
-    "prosemirror-keymap": "^1.1.4",
-    "prosemirror-model": "^1.14.3",
+    "prosemirror-keymap": "^1.1.5",
+    "prosemirror-model": "^1.16.1",
     "prosemirror-schema-list": "^1.1.6",
-    "prosemirror-view": "^1.20.1",
+    "prosemirror-view": "^1.23.6",
     "vue": "^2.6.14",
     "vuedraggable": "^2.24.3",
     "vuelidate": "^0.7.6",
```
Comment (afbora) [inline panel/package.json:32]: 0.7.7?

**Comment by bastianallgeier** [comment]
It works fine, but I still get the postcss warning.

File: panel/package.json (hunk 1 above)
Comment (distantnative) [reply panel/package.json:32]: @afbora can you find that there https://github.com/vuelidate/vuelidate/releases? I can't even find 0.7.6 - only 0.7.5 😆

**Comment by distantnative** [comment]
@bastianallgeier If I read it correctly, that originates from `vite-plugin-vue2` and no update that fixes it is yet available.

File: panel/package.json (hunk 1 above)
Comment (afbora) [reply panel/package.json:32]: Yes, I'm not sure why not listed but you can see on npm:

https://www.npmjs.com/package/vuelidate

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**